ios/HackerNews/Web/WebView.swift

```diff
@@ -10,23 +10,40 @@ import SwiftUI
 import UIKit
 import WebKit
 
+final class WebViewModel: ObservableObject {
```

Comment:
We should be using `@Observable` everywhere we can, I don't see a need for this to be `ObservableObject`